style/classless.css

```
@@ -10,212 +10,436 @@ Table of Contents:
+    sans-serif;
+  --font-c: 0.9em/1.4 "DejaVu Sans Mono", monospace;
+  --border: 1px solid var(--cmed);
+  --ornament: "‹‹‹ ›››";
+  /* foreground   | background color */
+  --cfg: #433;
+  --cbg: #fff;
+  --cdark: #888;
+  --clight: #f5f6f7;
+  --cmed: #d1d1d1;
+  --clink: #0f7dc2;
+  --cemph: #0086a9;
+  --clight-teal: #66cccc;
+  --cemphbg: #0881;
+}
+
+html[data-theme="dark"] {
+  /* foreground   | background color */
+  --cfg: #cecbc4;
+  --cbg: #252220;
+  --cdark: #999;
+  --clight: #333;
+  --cmed: #566;
+  --clink: #0f7dc2;
+  --cemph: #0086a9;
+  --clight-teal: #66cccc;
+  --cemphbg: #0b91;
 }
 
 @media (prefers-color-scheme: dark) {
-	html[data-theme='light'] {
-		/* foreground   | background color */
-		--cfg:   #cecbc4; --cbg:    #252220;
-		--cdark: #999;    --clight: #333;
-		--cmed:  #566;
-		--clink: #0F7DC2;
-		--cemph: #0086a9;    --cemphbg: #0b91;
-	}	
+  html[data-theme="light"] {
+    /* foreground   | background color */
+    --cfg: #cecbc4;
+    --cbg: #252220;
+    --cdark: #999;
+    --clight: #333;
+    --cmed: #566;
+    --clink: #0f7dc2;
+    --cemph: #0086a9;
+    --cemphbg: #0b91;
+  }
 }
 
 /* 2. Reset –––––––––––––––––––––––––––––––––––––––––––– */
 
 /* reset block elements  */
-* { box-sizing: border-box; border-spacing: 0; margin: 0; padding: 0;}
-header, footer, figure, table, video, details, blockquote,
-ul, ol, dl, fieldset, pre, pre > code, caption {
-	display: block;
-	margin: 0.5rem 0rem 1rem;
-	width: 100%;
-	overflow: auto hidden;
-	text-align: left;
+* {
+  box-sizing: border-box;
+  border-spacing: 0;
+  margin: 0;
+  padding: 0;
+}
+header,
+footer,
+figure,
+table,
+video,
+details,
+blockquote,
+ul,
+ol,
+dl,
+fieldset,
+pre,
+pre > code,
+caption,
+menu {
+  display: block;
+  margin: 0.5rem 0rem 1rem;
+  width: 100%;
+  overflow: auto hidden;
+  text-align: left;
+}
+video,
+summary,
+input,
+select {
+  outline: none;
 }
-video, summary, input, select { outline:none; }
 
 /* reset clickable things  (FF Bug: select:hover prevents usage) */
-a, button, select, summary { color: var(--clink); cursor: pointer; }
-
+a,
+button,
+select,
+summary {
+  color: var(--clink);
+  cursor: pointer;
+}
 
 /* 3. Base Style ––––––––––––––––––––––––––––––––––––––– */
-html { font-size: var(--rem); background: var(--cbg); }
+html {
+  font-size: var(--rem);
+  background: var(--cbg);
+  scroll-behavior: smooth;
+}
 body {
-	position: relative;
-	margin: auto;
-	max-width: var(--width);
-	font: var(--font-p);
-	color: var(--cfg);
-	padding: 3.0rem 0.6rem 0;
-	overflow-x: hidden;
+  position: relative;
+  margin: auto;
+  width: 100%;
+  font: var(--font-p);
+  color: var(--cfg);
+  padding: 3rem 0 0;
+  overflow-x: hidden;
+}
+body > footer {
+  margin: 10rem 0rem 0rem;
+  font-size: 90%;
+}
+
+/* grid style for content elements */
+article,
+header,
+footer,
+article > section {
+  display: grid;
+  grid-template-columns: 2fr min(var(--width), calc(100% - 96px)) 5fr;
+  column-gap: 48px;
+}
```

“塌陷”的样式来自这里。使用css样式表，将左右间距设为2:5，可随意修改。
如果觉得难看，可以随时改回1:1。
```suggestion
/* grid style for content elements */
article,header,footer,article > section {
  display: grid;
  grid-template-columns: 1fr min(var(--width), calc(100% - 96px)) 1fr;
  column-gap: 48px;
}
```

---

已经改正。